lib/payload.json

```diff
@@ -97,32 +102,40 @@
           "additionalProperties": false
         },
         "air": {
-          "type": "object",
-          "properties": {
-            "temperature": {
-              "description": "Air temperature (°C)",
-              "$ref": "#/definitions/temperature"
-            },
-            "relativeHumidity": {
-              "description": "Relative humidity (%)",
-              "$ref": "#/definitions/percentage"
-            },
-            "pressure": {
-              "type": "number",
-              "description": "Atmospheric pressure (hPa)",
-              "minimum": 900,
-              "maximum": 1100
+          "type": "array",
```

Comment:
OK so I only see this now; `air` gets changed from `object` to `array`? We can't do that, that breaks API.

The solution is to return an array from the normalizer. So if there are multiple measurements, there are multiple array items, each with their own `air` object.

Reply:
Ah i see, didnt know. Will get on it.